housePriceCalculator.js

```diff
@@ -0,0 +1,35 @@
+// Peter's house details
+const peterWidth = 8;
+const peterDepth = 10;
+const peterHeight = 10;
+const peterGardenSize = 100;
```

Comment:
I'd suggest you make those variables a bit more descriptive by adding `House` to them. eg.
```suggestion
const peterHouseWidth = 8;
const peterHouseDepth = 10;
const peterHouseHeight = 10;
const peterHouseGardenSize = 100;
```
Else, someone might think on first read it's all about the size of Peter 😀